### DISCUSSION TITLE
Looking For Help, Can't get Fyne Up And Running On Windows 11

### DISCUSSION THREAD

**Discussion by ghost** [fyne-io/fyne#4422] [Q&A]
Hello! I recently started learning Go and Fyne and wanted to try to get the demo working. I used TDM to install GCC and I believe all the environment variables are properly set to the best of my knowledge, and the setup tool agrees. I'm not sure if it matters, but when I installed Go, it installed to C:\Program Files\Go and created the two environment variables listed below (even though no folder for C:\Users\don\go was created, so I went ahead and made it. It looks like the fyne.exe is located in C:\Users\don\go\bin\. Not sure if that is correct or not. 

I am using these two pages to try to get up and running:
[https://developer.fyne.io/started/](https://developer.fyne.io/started/)
[https://developer.fyne.io/started/hello](https://developer.fyne.io/started/hello)

I have also watched this video:
[https://www.youtube.com/watch?v=S3T9l9QUa9I](https://www.youtube.com/watch?v=S3T9l9QUa9I)

Neither have led to a working Fyne project. Running or installing the demo does not work either. Also, I have run "go mod tidy".

Here are two pastes of the main.go and go.mod errors:
[https://pastebin.com/mtCuVBiN](https://pastebin.com/mtCuVBiN)
[https://pastebin.com/KHsWGtZQ](https://pastebin.com/KHsWGtZQ)

Terminal hangs indefinitely when trying to run.

![image](https://github.com/fyne-io/fyne/assets/33989414/4012db82-a3a4-4363-af58-d14a9ff34456)
![image](https://github.com/fyne-io/fyne/assets/33989414/01a1fc1c-3598-412d-89d5-b193968f3401)
![image](https://github.com/fyne-io/fyne/assets/33989414/01a17b91-cd46-4dd2-bc34-168845316bad)
![image](https://github.com/fyne-io/fyne/assets/33989414/ae72eb11-7a75-43a8-b527-b7549bc5f051)
![image](https://github.com/fyne-io/fyne/assets/33989414/eac71463-81b7-48fd-a2a6-789148a0d9b3)

**Comment by andydotxyz** [comment]
Run "go mod tidy" to make go download the dependencies for VSCode, it looks like you have two different environments possibly.

Secondly on windows the first build can take around 5 minutes! So get a cup of tea and see if it's finished.
